### PR TITLE
Add enum classes

### PR DESCRIPTION
**Pull request by unie31**
Not sure why github complains about missing newlines

### CODE REVIEW

File: forestdatamodel/constants/internal_const.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    GOAT_WILLOW = 20
+    COMMON_ASH = 21
+    KEDAR = 22
+    SERBIAN_SPRUCE = 23
+    OAK = 24
+    BIRD_CHERRY = 25
+    MAPLE = 26
+    CURLY_BIRCH = 27
+    WYCH_ELM = 28
+    UNKNOWN_CONIFER_SPECIES = 29
```
Comment (jpy-luke) [inline forestdatamodel/constants/internal_const.py:34]: coniferous

File: forestdatamodel/constants/internal_const.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    UNKNOWN_DECIDUOUS_SPECIES = 30
```
Comment (jpy-luke) [inline forestdatamodel/constants/internal_const.py:35]: CONIFEROUS. And the _SPECIES suffix is kinda redundant here

File: forestdatamodel/constants/internal_const.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    and listing all the distinct ones"""
+    PINE = 1
+    SPRUCE = 2
+    SILVER_BIRCH = 3
+    DOWNY_BIRCH = 4
+    ASPEN = 5
+    GREY_ALDER = 6
+    COMMON_ALDER = 7
+    OTHER_CONIFEROUS_SPECIES = 8
+    OTHER_DECIDUOUS_SPECIES = 9
```
Comment (jpy-luke) [inline forestdatamodel/constants/internal_const.py:14]: _SPECIES suffix is redundant here

File: forestdatamodel/constants/mela_const.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    ETELA_POHJANMAA = 7
+    KESKI_SUOMI = 8
+    POHJOIS_SAVO = 9
+    POHJOIS_KARJALA = 10
+    KAINUU = 11
+    POHJOIS_POHJANMAA = 12
+    LAPPI = 13
+
+class MelaForestManagementCategory(Enum):
+    '''NOTE: codes 4 and 5 do not exist in the Mela doc'''
```
Comment (jpy-luke) [inline forestdatamodel/constants/mela_const.py:84]: this comment is not necessary

File: forestdatamodel/constants/mela_const.py [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+class MelaTreeSpecies(Enum):
+    SCOTS_PINE = 1
+    NORWAY_SPRUCE = 2
+    SILVER_BIRCH = 3
+    DOWNY_BIRCH = 4
+    ASPEN = 5
+    ALDER = 6
+    OTHER_CONIFEROUS_SPECIES_THAN_1_OR_2 = 7
+    OTHER_DECIDUOUS_SPECIES_THAN_3_TO_6 = 8
```
Comment (jpy-luke) [inline forestdatamodel/constants/mela_const.py:108]: OTHER_CONIFEROUS and OTHER_DECIDUOUS is enough here

File: setup.py [hunk 6]
```diff
@@ -4,5 +4,5 @@
     name="forestdatamodel",
     description="Data classes and utilities for forest stand, tree strata and reference trees representation",
     version="0.1.2",
-    packages=["forestdatamodel"]
+    packages=["forestdatamodel", "forestdatamodel.constants"]
```
Comment (jpy-luke) [inline setup.py:7]: Why is this needed? The constants namespace package should be resolveable simply from the subdirectory.

**Review by jpy-luke — CHANGES_REQUESTED** [review]
Check individual comments.

Please rename `constants` package as `enums`
Please drop the _const suffix from the files.

Newline at end of file is defacto style standard for human readable files and gets nagged about by every proper software in existence, not just github. I think VSCode should be configurable so that it'll add newline upon save, or even better, run lint/code formatting upon save.

**Comment by jpy-luke** [comment]
I think squashing is fine in this case. Up to you.

File: setup.py (hunk 6 above)
Comment (unie31) [reply setup.py:7]: True, it was not needed

File: forestdatamodel/enums/internal.py [hunk 7]
```diff
@@ -0,0 +1,43 @@
+from enum import Enum
+
+
+class InternalTreeSpecies(Enum):
```
Comment (jpy-luke) [inline forestdatamodel/enums/internal.py:4]: I don't think the Internal prefix is needed with these. Just TreeSpecies is enough to convey that this is the general case.

**Review by jpy-luke — CHANGES_REQUESTED** [review]

**Review by jpy-luke — APPROVED** [review]
